modules/configuring-user-level-access.adoc

```diff
@@ -0,0 +1,47 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="configuring-user-level-access_{context}"]
+= Configuring user level access
+
+[role="_abstract"]
+To manage and modify the user level access you need to configure the RBAC section in Argo CD custom resource.
+
+.Procedure
+
+* Edit the `argocd` Custom Resource:
++
+[source,terminal]
+----
+$ oc edit argocd [argocd-instance-name] -n [namespace]
+----
+.Output
++
+[source,yaml]
+----
+metadata
+...
+...
+  rbac:
+    policy: 'g, rbacsystem:cluster-admins, role:admin'
+    scopes: '[groups]'
+----
++
+* Add the `policy` configuration to the `rbac` section and add the `name`, `email` and the `role` of the user: 
++
+[source,yaml]
+----
+metadata
+...
+...
+rbac:
+    policy: <name>, <email>, role:<admin>
+    scopes: '[groups]'
+----
+
+[NOTE]
+====
+Currently, RHSSO cannot read the group information of {gitops-title} users. So, RBAC must be configured at the user level.
```

Comment:
Suggestion, note requirement, but I'd put the action on the user:

`Currently, RHSSO cannot read the group information of {} users. Therefore, configure the RBAC at the user level.`